backend/chromeApiServer/src/main/java/com/canvas/service/helperServices/CanvasClientService.java

```diff
@@ -43,14 +42,14 @@ public class CanvasClientService {
     private static final Logger logger = LoggerFactory.getLogger(CanvasClientService.class);
 
 
-    /**
-     * Constructor with not arguments
-     */
-    public CanvasClientService() {
+    @Autowired
+    protected CanvasClientService(Environment env) {
+        this.canvasUrl = env.getProperty("canvas.url");
```

Comment:
This is intended to be a no-arg constructor that just has the okHttypClient. Undo the changes and leave as is, since your other constructor has the args

Reply:
This cannot be a no-arg constructor. env is a mandatory parameter required for starting the application. Since we have coverage this shouldn't be a problem.